derivation/src/test/scala/tofu/data/derived/DisplaySpec.scala

```diff
@@ -0,0 +1,146 @@
+package tofu.data.derived
+
+import cats.Eval
+import derevo.{derive, insertInstancesHere}
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import tofu.common.Display
+import tofu.common.Display._
+import tofu.data.derived.display
+
+class DisplaySpec extends AnyFunSpec with Matchers {
+  describe("derivation") {
+
+    @derive(display)
+    case class Bar(value: Int, another: String)
+
+    object Bar {
+      insertInstancesHere()
+    }
+
+    @derive(display)
+    case class Foo(bar: Bar, field: Double, xs: List[Int])
+
+    object Foo {
+      insertInstancesHere()
+    }
+
+    describe("simple cases") {
+      val bar = Bar(
+        3,
+        "abc"
+      )
+
+      val expectedBar =
+        """Bar{
```

Comment:
I don't like this default style with no space between the class name and the `{`